docs/publishing-to-rubygems.md

```diff
@@ -0,0 +1,29 @@
+# Publishing to RubyGems
+
+1. Checkout **master** and pull latest changes.
+
+2. Create and checkout a new branch (`release-[version-number]`).
```

Comment:
Is it worth mentioning to check the latest commits to make sure they're included in the Changelog? People unfamiliar with working in the repo can forget to add to the Changelog as Unreleased.

Reply:
Yep, good suggestion.